src/echo-common/test/poe-zone-tracker-service.test.ts

```diff
@@ -0,0 +1,257 @@
+import { expect, test, afterEach, beforeEach } from '@jest/globals'
+import { Observable, Subject } from 'rxjs'
+
+import { Tail } from 'tail'
+import path from 'path'
+import * as fs from 'fs'
+
+import { PoeClientLogService } from '../src/poe-client-log-service'
+
+import { PoeZoneDelta, PoeZoneTrackerService } from '../src/poe-zone-tracker-service'
+import { EchoDirService, PoeCharacterService, SmartCacheEvent } from '../src'
+import { PoeCharacter } from 'sage-common'
+import { GggApi, GggHttpUtil } from 'ggg-api'
+import { SmartCacheResultEvent } from '../src/smart-cache'
+
+jest.setTimeout(30000)
+
+let file: string
+let tail: Tail
+
+beforeEach(() => {
+  const filename = Math.floor(Math.random() * 999999999999) + '.txt'
```

Comment:
if there isn't a reason these need to be numbers it would probably be better to use https://www.npmjs.com/package/uuid I think we already have this as a dep everywhere.

Reply:
I added and now use the uuid package, it was not originally used in echo-common